xilinx_test/kernels/DDRExplicit.cpp

```diff
@@ -0,0 +1,13 @@
+/// @author    Jannis Widmer (widmerja@ethz.ch)
+/// @copyright This software is copyrighted under the BSD 3-Clause License.
+
+#define DATA_SIZE 1024
```

Comment:
Use `constexpr`

Reply:
Sorry about that. Guess I should learn how to use a search ;)